CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Is a `CMakeLists.txt` necessary for this project? I ask because the file itself seems fairly straightforward, you may be able to compile it directly in your Dockerfile (something like `clang -fsanitize=fuzzer -o my_printf_fuzzer my_printf_fuzzer.c`?)

Reply:
The training on CMake was really just to familiarize people with common build systems, not to necessarily imply that one was required to create a fuzz target.

Reply:
I have improved a lot since this submission and will make some adjustments. Do I just submit a new pull request with the updated repository?

Reply:
Yes, if you'd like to submit a different pull request, simply create a new PR and I will close this one and look at the new one.